File: Nível 1/2006 - Identificando o Chá/main.py
valor = int(input("A: "))
resposta = input("B: ").split(" ")
competidores = 0

if int(resposta[0]) == valor:
    competidores += 1

if int(resposta[1]) == valor:
    competidores += 1

if int(resposta[2]) == valor:
    competidores += 1

if int(resposta[3]) == valor:
    competidores += 1

if int(resposta[4]) == valor:
    competidores += 1

print(competidores)

File: Nível 1/2006 - Identificando o Chá/main.lua
valor = tonumber(io.read())
resposta = io.read()
x1, x2, x3, x4, x5 = resposta:match("(%S+)%s+(%S+)%s+(%S+)%s+(%S+)%s+(%S+)")
x1, x2, x3, x4, x5 = tonumber(x1), tonumber(x2), tonumber(x3), tonumber(x4), tonumber(x5)
competidores = 0

if x1 == valor then 
    competidores = competidores + 1
end

if x2 == valor then 
    competidores = competidores + 1
end

if x3 == valor then 
    competidores = competidores + 1
end

if x4 == valor then 
    competidores = competidores + 1
end

if x5 == valor then 
    competidores = competidores + 1
end

print(competidores)